feature-libs/cart/quick-order/components/quick-order/quick-order.component.html

```diff
@@ -1,136 +1,165 @@
-  <cx-message
-    *ngFor="let deletedEntry of deletedEntries | keyvalue"
-    (buttonAction)="undoDeletion(deletedEntry.value)"
-    (closeMessage)="clearDeletion(deletedEntry.value)"
-    [actionButtonText]="'quickOrderList.undo' | cxTranslate"
-    [text]="
-      'quickOrderList.productWasDeleted'
-        | cxTranslate: { name: deletedEntry.value.product?.name }
-    "
-    [type]="globalMessageType.MSG_TYPE_CONFIRMATION"
-    class="quick-order-deletions-message"
-  >
-  </cx-message>
-</ng-container>
+  <ng-container *ngIf="softDeletedEntries$ | async as deletedEntries">
+    <cx-message
+      *ngFor="let deletedEntry of deletedEntries | keyvalue"
+      (buttonAction)="undoDeletion(deletedEntry.value)"
+      (closeMessage)="clearDeletion(deletedEntry.value)"
+      [actionButtonText]="'quickOrderList.undo' | cxTranslate"
+      [actionButtonMessage]="
+        'quickOrderList.revokeUndo'
+          | cxTranslate: { name: deletedEntry.value.product?.name }
+      "
+      [text]="
+        'quickOrderList.productWasDeleted'
+          | cxTranslate: { name: deletedEntry.value.product?.name }
+      "
+      [type]="globalMessageType.MSG_TYPE_CONFIRMATION"
+      class="quick-order-deletions-message"
+    >
+    </cx-message>
+  </ng-container>
 
-<ng-container *ngIf="errors$ | async as errors">
-  <cx-message
-    *ngIf="errors.length"
-    (closeMessage)="clearErrors()"
-    [accordionText]="'quickOrderList.errors.reviewErrors' | cxTranslate"
-    [text]="'quickOrderList.errorProceedingToCart' | cxTranslate"
-    [type]="globalMessageType.MSG_TYPE_ERROR"
-    class="quick-order-errors-message"
-  >
-    <ul class="quick-order-errors">
-      <li *ngFor="let error of errors" class="quick-order-error-item">
-        <span>
-          {{
-            'quickOrderList.errors.productIsOutOfStock'
-              | cxTranslate
-                : {
-                    name: error.entry.product.name,
-                    code: error.entry.product.code
-                  }
-          }}
-        </span>
-      </li>
-    </ul>
-  </cx-message>
-</ng-container>
+  <ng-container *ngIf="errors$ | async as errors">
+    <cx-message
+      *ngIf="errors.length"
+      (closeMessage)="clearErrors()"
+      [accordionText]="'quickOrderList.errors.reviewErrors' | cxTranslate"
+      [text]="'quickOrderList.errorProceedingToCart' | cxTranslate"
+      [type]="globalMessageType.MSG_TYPE_ERROR"
+      class="quick-order-errors-message"
+    >
+      <ul class="quick-order-errors">
+        <li *ngFor="let error of errors" class="quick-order-error-item">
+          <span>
+            {{
+              'quickOrderList.errors.productIsOutOfStock'
+                | cxTranslate
+                  : {
+                      name: error.entry.product.name,
+                      code: error.entry.product.code
+                    }
+            }}
+          </span>
+        </li>
+      </ul>
+      <div aria-live="polite" class="cx-visually-hidden">
```

Comment:
I think this `aria-live="polite"` attribute is not necessary here, as we have all messages wrapped into another aria-live attribute.